### ISSUE TITLE
Moving window with a variable size

### ISSUE THREAD

**Issue by keithbennett** [Warwick-Plasma/epoch#44]
In GitLab by Zhongwei on 01 Jul 2018, 10:21  (GitLab issue GL#1778)

### Description

When I study the wakefield acceleration, the moving window module is quite helpful. Instead, if we want to study the acceleration in the local position and save CPU time. A simulation box that keeps shrinking in size will be helpful as well.

### Links / references

Similar to that you have done for the moving window. In a moving window, you change both sides of the simulation box.
Reducing box is an extension. I.e., Fix one side and change another side of the box.
In other words, just add an option to switch on and off the moving boundary at xmax, xmin, ymax, ymin, zmax, zmin. 
I hope it is possible and look forward to your enhancement of epoch. Thank you very much.
![box](../raw/37c47bd3c9805e46ecfa4681221ec19dd3629922/uploads/a567b97b57e2aa7f9c6066a4b8ea75bb/box.png)
